README.md

```diff
@@ -40,6 +45,17 @@ DeployGate SDK uses `ContentProvider` to initialize itself so you need to remove
 </application>
 ```
 
+And you also need to add `DeployGate.install()` to your implementation.
```

Comment:
```suggestion
And also, you need to call `DeployGate#install` in your Application class, ContentProvider or AndroidX Startup Initializer.
```

Reply:
Sorry, I have misunderstood.
SDK is didn't supported  to call method with named arguments.